manager/api/serializers.py

```diff
@@ -170,17 +175,21 @@ def get_time_data(self, token) -> dict:
             - utc: current time in UTC scale as a unix timestamp (seconds)
             - tai: current time in UTC scale as a unix timestamp (seconds)
             - mjd: current time as a modified julian date
-            - sidereal_summit: current time as a sidereal_time w/respect to the summit location (hourangles)
-            - sidereal_summit: current time as a sidereal_time w/respect to Greenwich location (hourangles)
-            - tai_to_utc: The number of seconds of difference between TAI and UTC times (seconds)
+            - sidereal_summit: current time as a sidereal_time
+            w/respect to the summit location (hourangles)
+            - sidereal_summit: current time as a sidereal_time
+            w/respect to Greenwich location (hourangles)
+            - tai_to_utc: The number of seconds of difference
+            between TAI and UTC times (seconds)
         """
         return utils.get_times()
 
     @swagger_serializer_method(serializer_or_field=serializers.JSONField())
     def get_config(self, token) -> Union[dict, None]:
```

Comment:
Please don't use `typing.Union` but instead do this: `dict | None`. Change this everywhere please.